workflows/mcmicro.nf

```diff
@@ -88,6 +89,17 @@ workflow MCMICRO {
     }
     */
 
+    // Run Coreograph
+    if (params.tma_dearray) {
+        COREOGRAPH(ASHLAR.out.tif)
+        COREOGRAPH.out.cores
+            .transpose()
```

Comment:
Good use of `transpose` here!

Reply:
Thank you!
I'm starting to get a little more comfortable with these operators.